### ISSUE TITLE
Consider presenting Download and Export options differently

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/earthworks#646]
(I considered submitting this to the GeoBlacklight repo, since it seems like a common pattern other institutions might benefit from changing, but there is enough variation among institutions that I worry it might be too big of a change to impose in GeoBlacklight. @mejackreed will probably have a better sense of whether we do this locally or suggest for GeoBlacklight, if we decide to make a change.)

I'm concerned about the amount of sidebar space that is used by the download and export options, especially if we introduce some form of relations box in the sidebar. I'm also concerned about having a handful of similarly-looking boxes in the sidebar, some of which might be more important than others. (I'd like to rethink the Tools box as well, but will create a separate ticket for that.)

I also have some concerns about how the export and download affordances are presented. Those details combined with the use of boxes to contain everything make the sidebar kind of large and busy, in my opinion.

---

<img width="477" alt="Screen Shot 2020-06-12 at 1 30 57 PM" src="https://user-images.githubusercontent.com/101482/84545264-bf7d3a00-acb3-11ea-8334-e2dec6e3b019.png">

---

### A - Downloads
- As best I can tell, items in Earthworks always, or almost always, only have one download available. Using a box to display one button feels unnecessary to me (although I get it was probably done for consistency).

- Again, there might be exceptions, but every download example I saw used a label that started with "Original." I wonder if the "Original" part is really necessary?

### B - Export formats
- This box often has several items in it, but I still wonder if placing the buttons in a box makes sense.

- The button labels are always the same: "Export" (probably not great for accessibility) and the export format is presented outside of the button. This feels necessary to me, and adds to the vertical space required (at least at some viewport widths).

### Proposed revision
<img width="880" alt="Screen Shot 2020-06-12 at 2 00 04 PM" src="https://user-images.githubusercontent.com/101482/84545929-0fa8cc00-acb5-11ea-9ac0-1edd0437c706.png">

- Remove the Downloads and Export formats boxes

- Eliminate the use of "Original" in the download label

- For both download and export, use a single button to present the applicable options

- If there is more than one option, use a split button with the different download/export options

- I personally prefer the small size of the button (31px tall in my mockups versus 38px in the first screenshot of the current app), since these are secondary tasks and to reduce the vertical size of the sidebar, but if we use the split button approach this is less of an issue.

**Comment by jvine** [comment]
Tagging with a11y because it covers #659

**Comment by mejackreed** [comment]
@ggeisler an additional consideration here is the hopeful expansion of the actual "Downloads" to handle multiple downloads. An example of how data could look would be something like this:

https://geo.nyu.edu/catalog/nyu-2451-38644

![Screen Shot 2020-06-16 at 10 30 21 AM](https://user-images.githubusercontent.com/1656824/84801444-6c113180-afbc-11ea-98bf-81427110f59b.png)

This is an intended feature in GeoBlacklight but might be something we will need to eventually consider.

**Comment by ggeisler** [comment]
@mejackreed Sure, I was taking that possibility into account in my suggestion above:

> For both download and export, use a single button to present the applicable options

> If there is more than one option, use a split button with the different download/export options

In other words, unless there is a good reason not to, we would treat the Export and Download options in the same way, using a split button if there are multiples in the category. So the NYU example above would present those download options with a split button:

Download >
  LAZ (point-cloud)
  LAZ (full-waveform)
  GeoTIFF (geo-referenced RGB)
  GeoTIFF (geo-referenced CIR)
  JPG (oblique photos)

(where button label is "Download" and the lines below are the menu options)

**Comment by mejackreed** [comment]
Great thanks! One thing I just want to call out is there a way for users to know about the data that is available to download? maybe that is less important just that they can download something?

**Comment by ggeisler** [comment]
If what you're getting at is by putting the download options into a split button, we're hiding the specific format options available because the user now has to actually select the button to see them, then yes, that's definitely the tradeoff. Immediate visibility of the options versus compactness of the sidebar.

My feeling is it is enough that the user knows downloads are available; if that user is actually interested in downloading something, then I assume they'll be motivated to click the button to see what formats they can download.

I acknowledge this is a tradeoff, though. I'm just concerned about simplifying the sidebar so it is easier to scan and doesn't detract from the main content area by potentially being very busy with a lot of buttons.

I'll discuss this more with @jvine in the context of related things, like the Tools box and the new relationships box we're hoping to add.

**Comment by ggeisler** [comment]
@mejackreed You asked earlier if this was ready to work on. Yes, with one addition to the design above. 

The strategy outlined above for the download and export button labeling and using a single button/multi-select button is still valid.

A new related suggestion is that the Download and Export buttons, when present for an item, should go into a sidebar box/card labeled "Services".

I'll be posting some more detailed tickets related to the Services box idea soon, but the basic idea is that we want to group Download, Export, Web services, and Open in Carto (and any other services, if there are any) into this box:

<img width="252" alt="Screen Shot 2020-06-30 at 3 04 35 PM" src="https://user-images.githubusercontent.com/101482/86181959-d4dfca00-bae3-11ea-9346-f251f04cc3e5.png">

This box would go below any Data Relations boxes.

**Comment by thatbudakguy** [comment]
It looks like the initial portion of this has already been implemented in GeoBlacklight v4 — the downloads are stacked inside a collapsible button. Here's an example on a fresh install of GeoBlacklight v4:

![Screenshot 2023-03-29 at 13 41 04](https://user-images.githubusercontent.com/4924494/228662523-13e2c312-7237-4598-87b6-34b0e0ab91c5.png)

Rather than trying to backport this UI change to v3.x, I think we should consider this blocked until we can upgrade Earthworks to Geoblacklight v4+. Then, we can address the other change Gary proposed of moving download/export/web services into a single panel.

**Comment by dbranchini** [comment]
Addressed by new designs:

![Image](https://github.com/user-attachments/assets/d01fe1e6-c6b1-47a9-b10a-cbaf3550f3ac)

https://www.figma.com/proto/nkgg905s5FyXoyPZjVMco0/EarthWorks?page-id=0%3A1&node-id=971-31966&viewport=-7294%2C1192%2C0.56&t=kYki5n8Z1jZKPmeu-1&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=971%3A31966
